app/gateway-manager/data-plane-reference.md

```diff
@@ -13,24 +12,213 @@ works_on:
+
+{{site.konnect_short_name}} provides Data Plane node installation scripts for various platforms. 
+These Data Plane nodes are configured to run in your {{site.konnect_short_name}} environment. Alternatively, {{site.konnect_short_name}} offers fully-managed Data Planes through [Dedicated Cloud Gateways](/dedicated-cloud-gateways).
+
+## Supported installation options
+
+{{site.konnect_short_name}} supports the following installation options:
+
+{% table %}
+columns:
+  - title: Setup Type
+    key: type
+  - title: Platforms
+    key: platforms
+rows:
+  - type: Standard setup
+    platforms: macOS (ARM), macOS (Intel), Windows, Linux (Docker)
+  - type: Advanced setup
+    platforms: Linux, Kubernetes, AWS, Azure, Google Cloud
+{% endtable %}
+
+
+## Forward proxy support
+
+{{site.konnect_product_name}} supports using non-transparent forward proxies to connect your {{site.base_gateway}} Data Plane with the {{site.konnect_short_name}} Control Plane. See the [Forward proxy connections](/gateway/cp-dp-communication/) {{site.base_gateway}} documentation for more information.
+
+
+## Upgrade Data Planes
+
+Self-managed Data Plane nodes can be upgraded to a new {{site.base_gateway}} by initializing new nodes before decommissioning old ones. This method ensures high availability, allowing the new node to commence data processing prior to the removal of the old node. Managed nodes are upgraded automatically after selecting the new version of {{site.base_gateway}}. We recommend running one major version (2.x or 3.x) of a Data Plane node per Control Plane, unless you are in the middle of version upgrades to the Data Plane. Mixing versions may cause [compatibility issues](/konnect-platform/konnect-compatibility/).
+
+To upgrade a Data Plane node to a new version, follow these steps:
+
+{% navtabs "Upgrade" %}
+{% navtab "Dedicated Cloud Gateways" %}
+
+Using both the `control_plane_id`, `cloud_gateway_network_id`, and the desired [`version`](/konnect-platform/konnect-compatibility//) you can use the API to upgrade a Data Plane node:
+<!-- vale off -->
+
+{% control_plane_request %}
+url: /v2/cloud-gateways/configurations
+status_code: 201
+method: PUT
+headers:
+    - 'Accept: application/json'
+    - 'Content-Type: application/json'
+    - 'Authorization: Bearer $KONNECT_TOKEN'
+body:
+    control_plane_id: $CONTROL_PLANE_ID
+    version: 3.10
+    control_plane_geo: ap-northeast-1
+    dataplane_groups:
+      - provider: aws 
+      - region: $REGION
+      - cloud_gateway_network_id: $CLOUD_GATEWAY_NETWORK_ID
+      - autoscale: 
+        - kind: autopilot
+        - base_rps: 100
+{% endcontrol_plane_request %}
+
+<!-- vale on -->
+
+
+Kong performs a rolling upgrade of the fully-managed Data Plane nodes. This is a zero downtime upgrade because Kong synchronizes the Data Plane with load balancer registration and de-registration and gracefully terminates the old Data Plane nodes to reduce the impact on the ongoing traffic.
```

Comment:
```suggestion
{{site.konnect_short_name}} performs a rolling upgrade of the fully-managed Data Plane nodes. 
This is a zero downtime upgrade because {{site.konnect_short_name}} synchronizes the Data Plane with load balancer registration and de-registration and gracefully terminates the old Data Plane nodes to reduce the impact on the ongoing traffic.
```